contracts/staking/stakeManager/StakeManagerStorage.sol

```diff
@@ -33,7 +34,7 @@ contract StakeManagerStorage is ProxyStorage, Lockable, RootChainable {
     uint256 public auctionPeriod = dynasty / 4; // 1 week in epochs
     bytes32 public accountStateRoot;
 
-    // on dynasty update certain amount of cooldown period where there is no validator auction
+    // on dynasty update for a certain amount of cooldown period where there is no validator auction
```

Comment:
Not clear to me what this comment means.

Reply:
fixed